src/Acuminator/Acuminator.Utilities/CodeAnalysisSettings.cs

```diff
@@ -11,17 +11,51 @@ namespace Acuminator.Utilities
 	public class CodeAnalysisSettings
 	{
 		public static CodeAnalysisSettings Default => 
-			new CodeAnalysisSettings(recursiveAnalysisEnabled: true);
+			new CodeAnalysisSettings(
+				recursiveAnalysisEnabled: true, 
+				isvSpecificAnalyzersEnabled: false);
 
 		public CodeAnalysisSettings()
```

Comment:
@dropsonic maybe the setting values shoud be the same as in the instance returned by the Default  property.
It feels inconsistent to have `new CodeAnalysisSettings()` with settings different from the  `CodeAnalysisSettings .Default`
If you plan to use it like VS Workspace Options then it could be like this `CodeAnalysisSettings .Default.WithRecursiveAnalysisEnabled()`. The default constructor feels redundant. Instead you could make the private constructor to be public

Reply:
@SENya1990 default constructor is used by the derived class which doesn't store those values - it gets them from the Options page

Reply:
But this is not the only one possible option to create default code analysis settings. If you want only derived classes to use the default constructor then it can be `protected`

Reply:
@SENya1990 default constructor is used in the unit tests

Reply:
It is inconsistent to have the options set by the default constructor to be different from the ones set by the Default property. Imho they should be the same